src/services/item/repository.ts

```diff
@@ -234,15 +235,29 @@ export const ItemRepository = AppDataSource.getRepository(Item).extend({
       .getRawOne();
   },
 
-  async getOwn(memberId: string): Promise<Item[]> {
-    return this.createQueryBuilder('item')
+  async getOwn(
+    memberId: string,
+    page: number,
+    name: string,
+    all: boolean,
+    limit: number,
```

Comment:
I think it might be cool to wrap these properties (`page`, `all`, `limit` in something like a `PageRequest` or something, that could be useful for future pagination too, but this is not absolutely needed right now.

Then for filters and sorting, I don't know if I would try to put it inside this `PageRequest` with arbitrary strings somehow.

Right now, I think you could still make `name` argument optional if it's not required when calling `getOwn`